src/the-bureau/index.html

```diff
@@ -184,8 +186,8 @@ <h1 class="h2">CFPB leadership</h1>
                              alt="Director Richard Cordray">
                     </div>
                     <div class="media_body">
-                        <h1 class="h6">Director</h1>
-                        <h2 class="bureau-bio_name">Richard Cordray</h2>
+                        <span class="h6">Director</span>
```

Comment:
Why set his position as the heading on `/the-bureau/bureau-structure/`, but his name as the heading here?